examples/example_for_updating_metadata.py

```diff
@@ -65,6 +100,9 @@ def add_values_dataset_description(dataset_description):
     # code_parameters = dataset.get_metadata(category="code_parameters")
     # code_description = dataset.get_metadata(category="code_description")
 
+    des_schema = schema.get_schema("dataset_description")
```

Comment:
would be great if can also do `schema.get_schema("dataset_description", "Subtitle")`